packages/components/src/components/hds/time/range.hbs

```diff
@@ -0,0 +1,14 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+{{! IMPORTANT: we need to add "squishies" here (~) because otherwise the whitespace added by Ember causes extra space around the time element - See https://handlebarsjs.com/guide/expressions.html#whitespace-control }}
+<span class="hds-time hds-time-range" ...attributes>
```

Comment:
as we have `hds-time--single`, I believe this should use the same modifier convention
```suggestion
<span class="hds-time hds-time--range" ...attributes>
```